test/unit/Input.test.js

```diff
@@ -51,61 +51,61 @@ describe('Props can be set properly', () => {
 
     test('props.id is set as the <input> id', () => {
         // test if id is in the actual HTML string
-        const inputTag = input.render().children();
+        const inputTag = input.render();
```

Comment:
What's the difference between `render()` and `render().children()` ? Does the change to the latter have to do with the React 16 upgrade?

Reply:
It has to do with the Enzyme upgrade.